src/api/handlers/guild.ts

```diff
@@ -769,13 +769,23 @@ export function getGuild(guildID: string, counts = true) {
 }
 
 /** Returns the guild template if it exists */
+export async function getTemplate(templateCode: string) {
+  const result = await RequestManager.get(
+    endpoints.GUILD_TEMPLATE(templateCode),
+  ) as GuildTemplate;
+  const template = await structures.createTemplate(result);
+  return template;
+}
+
+/** 
+ * Returns the guild template if it exists 
+ * @deprecated will get removed in v11 use `getTemplate` instead
+ */
 export function getGuildTemplate(
   guildID: string,
   templateCode: string,
 ) {
-  return RequestManager.get(
-    `${endpoints.GUILD_TEMPLATES(guildID)}/${templateCode}`,
-  ) as Promise<Template>;
+  return getTemplate(templateCode);
```

Comment:
Do not change the implementation of the handler.

Reply:
Implementation is not changed only fixed. The endpoint does not exist.